common/country_types/hfe_country_types.txt

```diff
@@ -209,17 +247,41 @@ hfe_drone = {
 					base = 33
 				}
 			}
+#			ancient_drone_station = {
+#				min = 1
+#				max = 4
+#			}
+			ancient_mining_drone = {
```

Comment:
This country has no access to this ship sizes, remove it.

Reply:
They do have access to these ship sizes. I made sure that they got the Global Designs.

Reply:
If it have access to this ship sizes, it shoulnt

Reply:
Yes they should. The Harvesting Marvel is consistent entirely of Ancient Mining Drones.

Reply:
This is not related to vanilla mining drones.
Just share assets.

Reply:
The Harvesting Marvel IS an Ancient Mining Drone entity. I specifically tied them to the other Ancient Mining Drones in both text and story. I did this a long, long time ago, even before this Branch. I even had them set as Friendly to the Vanilla rendition ones!